src/openforms/js/components/form/edit/tabs.js

```diff
@@ -321,28 +322,82 @@ const PREFILL = {
   ],
 };
 
+const LANGUAGES = jsonScriptToVar('languages', {default: []});
+const tabComponents = LANGUAGES.map(([languageCode, _label]) => {
+  return {
+    key: languageCode,
+    label: languageCode.toUpperCase(),
+    components: [
+      {
+        type: 'datagrid',
+        input: true,
+        label: 'Translations',
+        key: `of-translations.${languageCode}`,
+        tooltip: 'Translations for literals used for this field',
+        weight: 10,
+        reorder: false,
+        components: [
+          {
+            label: 'Literal',
+            key: 'literal',
+            input: true,
+            unique: true,
+            type: 'textfield',
+          },
+          {
+            label: 'Vertaling',
```

Comment:
```suggestion
            label: 'Translation',
```
we'll manage that through our own static translations for formio :)